zerver/lib/import_realm.py

```diff
@@ -1018,6 +1019,10 @@ def do_import_realm(import_dir: Path, subdomain: str, processes: int = 1) -> Rea
         if "zerver_usergroup" not in data:
             set_default_for_realm_permission_group_settings(realm)
 
+    if settings.PUSH_NOTIFICATION_BOUNCER_URL is not None:
+        event = {"type": "send_realms_only_to_push_bouncer"}
+        queue_json_publish("deferred_work", event)
```

Comment:
A few suggested tweaks here:
* Let's include the `realm_id` in the event -- we may not bother doing it now, but in theory we only need to send the one realm to the bouncer and it'd be nice to not have to do compatibility code for a format migration if we decide to do that later.
* Let's call the event `register_realm_with_push_bouncer`. The function that happens to implement that right now is a call to `send_realms_only_to_push_bouncer`, but that's sorta not what we're doing at a high level here, and renaming this `type` will be annoying.

Reply:
Also I guess the whole block of 3 lines we can put in `zerver/lib/remote_server.py` in a function called `enqueue_register_realm_with_push_bouncer`, and put the comments/etc. in there, so that we just have a one-line call to that function passing the `realm_id` in both this and the `create_realm` code path.

(I'm mostly worried about the comments only being in one place or being duplicated).

Reply:
@timabbott How about `enqueue_register_realm_with_push_bouncer_if_needed`? (Because it's conditional on using the bouncer)

Pushed the requested changes. Also replaced `if settings.PUSH_NOTIFICATION_BOUNCER_URL is not None` with `if uses_notification_bouncer()`, because that's what that function is for.